_posts/2017-10-13-jb-in-da-house.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "J.B. Rainsberger in da house"
+date: 2017-10-13 14:03:00 +0200
+categories: learn product 
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 3
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-13-jbhc17/poster.jpg
+---
+
+It started in May, that we thought about inviting [J.B. Rainsberger][jbrains] to HolidayCheck. Yeah, there was some bias from my side, since I had the pleasure to have learned from him a couple years ago. So I was all in. I just had to convince my colleagues and the company to pay for it. And it went quite smooth. HolidayCheck is quite committed to improve continuously.
+
+[jbrains]: https://twitter.com/jbrains
+
+## 7:26min
+
+You have never heard of “7 minutes 26 seconds”? Now you did! It is a short and inspiring [lightning talk][7-26-talk] by J.B. called “7 minutes, 26 seconds, and the Fundamental Theorem of Agile Software Development”. Here at HolidayCheck people refer to it as the “Accidental Complication“ talk. I think it inspired many people here to be as excited about having J.B. around. 
+
+It’s not even necessary mentioning that this event won’t be a pure TDD session, or a programmers-only workshop. Even though I had thought of it like that before. Melanie, who is one of our product owners, jumped in early to help to make [#jbhc17][jbhc17] happen. And it became clear right away that this event will not only be teaching us developers, but also the product people about the tools we need to deliver value as fast as possible to our Urlauber (how we call our users).
+
+[7-26-talk]: https://vimeo.com/79106557
+[jbhc17]: https://twitter.com/search?q=%23jbhc17&src=typd
+
+## Remove bottlenecks
+
+The topic “Move the bottleneck out of product development” makes it very clear that our goal is to lead the market. Once we have no bottlenecks in product development. Once we deliver the best thing for our Urlauber. Once we run at high speed. Once we have no useless friction slowing us down, we will be able to lead the market. We will deliver the best value. And we will have even more fun and be the best Urlauber company. And we get where we want to be: we will have to look for the bottlenecks in the market, not in our company anymore. We will be able to try new business models, we will try to innovate, build, measure and learn at high speed.
+That is why we invited J.B. to learn where we can improve in those areas.
+
+## How?
+
+On Monday we will start with a keynote, have a nice lunch and will then be able to pick the right first bottleneck per team and practice the tools to fix it. J.B. will be leading us and be available to help, teach and practice with us.
+On Tuesday we will do a full-time workshop, continue applying the learnings and get a deeper insight on “Value-Driven Product Development”.
+
+## You would also like to?
```

Comment:
`Would you like to do it as well?` <-- is that what you meant?

Reply:
i want to say "you also like to learn from JB?" => join HC :)

Reply:
Then I would rephrase it to what I suggested. Because yours lacks verb, e.g. 'doing' hence 'do it'. I am not expert in english, but I guess it will sound better with than without it.